api/src/serge/main.py

```diff
@@ -0,0 +1,97 @@
+import asyncio
+import logging
+
+from fastapi import FastAPI
+from fastapi.middleware.cors import CORSMiddleware
+from fastapi.staticfiles import StaticFiles
+from starlette.responses import FileResponse
+
+from serge.routers.chat import chat_router
+from serge.routers.model import model_router
+from serge.utils.initiate_database import initiate_database, Settings
+from serge.dependencies import convert_model_files
+
+# Configure logging settings
+logging.basicConfig(
```

Comment:
Thoughts on using loguru instead?

https://github.com/Delgan/loguru

Reply:
Didn't know about it, it looks really cool. Definitely gonna use it, unless you want to do a PR for it